include/openssl/err.h

```diff
@@ -270,7 +257,9 @@ int ERR_load_ERR_strings(void);
 
 DEPRECATEDIN_1_1_0(void ERR_remove_thread_state(void *))
 DEPRECATEDIN_1_0_0(void ERR_remove_state(unsigned long pid))
-ERR_STATE *ERR_get_state(void);
+
+typedef struct err_state_st ERR_STATE;
+#define ERR_get_state() NULL
```

Comment:
It would be better if it could be marked deprecated. The best way to do this is to keep providing it as an actual function that returns NULL, and have this line here:

``` C
DEPRECATEDIN_3(ERR_STATE *ERR_get_state(void))
```

Reply:
You forgot this spot